terraform/variables.tf

```diff
@@ -251,9 +234,19 @@ variable "tier" {
 variable "database_version" {
   description = "The MySQL, PostgreSQL or SQL Server (beta) version to use."
   type        = string
-  default     = "MYSQL_5_7"
+  default     = "MYSQL_5_6"
```

Comment:
why not 5.7? just curious, in AWS 5.6 is deprecated and users will be forced to upgrade till 1st of September 2021.